modules/create_timepoint/php/NDB_Form_create_timepoint.class.inc

```diff
@@ -102,13 +102,22 @@ class NDB_Form_create_timepoint extends NDB_Form
         // label rules
         if ($visitLabelAdded) {
             $this->addRule('visitLabel', 'Visit label is required', 'required');
+
+	    // List of sites
+            $list_of_sites = Utility::getSiteList();
+            $psc_labelOptions = array(null => '');
+            foreach($list_of_sites as $psc => $siteAlias) {
+                    $psc_labelOptions[$psc] = $siteAlias;
+            }
+            $this->addSelect('psc', 'Site', $psc_labelOptions);
```

Comment:
wasn't the decision in the imaging meeting to add it iff the user is part of more than 1 site?